doc-content/enterprise-only/openshift/template-deploy-ldap-proc.adoc

```diff
@@ -16,7 +16,7 @@ You must create a user with the username and password configured in the secret f
 * You started the configuration of the template, as described in {startlink}.
 
 .Procedure
-. Set the `AUTH_LDAP*` parameters of the template. These parameters correspond to the settings of the `LdapExtended` Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/login_module_reference/#ldapextended_login_module[LdapExtended login module].
+. Set the `AUTH_LDAP*` parameters of the template. These parameters configure LDAP authentication using the Elytron subsystem of {EAP}. For mode information about the  instructions about using the Elytron subsystem of {EAP} with LDAP, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/how_to_configure_identity_management/elytron_secure_apps#elytron_ldap_auth_app[Configure Authentication with an LDAP-Based Identity Store].
```

Comment:
Did you mean for more information (not mode)?

Assuming so, I suggest:P,
or more information about  using the Elytron subsystem of {EAP} with LDAP,